spec/controllers/friendships_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+describe FriendshipsController do 
+  let(:joe) { Fabricate(:user) }
+  let(:jane) { Fabricate(:user) }
+  let(:deshawn) { Fabricate(:user) }
+  let(:john) { Fabricate(:user) }
```

Comment:
I didn't think about moving it up this high (my initial thought would have been within the various routes [index, destroy...]. I think this is okay here though mostly because they are all just base users with no extras so keeping them in your mind is easy. Just be careful not to move the abstraction too far away from the implementation because that becomes harder to read.